WcaOnRails/app/models/registration.rb

```diff
@@ -263,6 +263,16 @@ def self.wcif_json_schema
     end
   end
 
+  validate :forcing_competitors_to_add_comment
+  private def forcing_competitors_to_add_comment
+    if !competition || !competition.force_comment_in_registration
```

Comment:
I see you took this from `cannot_register_for_unqualified_events`, but I think the ruby-way of doing it would be `competition&.force_comment_in_registration.blank?` 
The &. operator is a 'safe access' Operator and returns nil if competition is nil. I am also fairly new to ruby, but that's how it's done in a lot of other code I saw

Reply:
done, still works locally after that change